web-components/src/components/charts/charts.stories.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 
-import moment from 'moment';
```

Comment:
moment has a notoriously large bundle size compared to other options - I was already using `date-fns` so pulled it in here, but the API is slightly different than moment. 

We could switch `date-fns` for [day.js](https://day.js.org/) if we want to keep `moment`'s API

Reply:
Ah. I'm fine with `date-fns`. Looks like it has the same capabilities. However, we may have `moment` in the bundle regardless. It may be a dependency for some other packages - I'm not sure.

Reply:
@mhagel I saw it in the yarn.lock as well, and it very well might already be pulled in as you say, but I also thought it might be parsed out through tree shaking (moment isn't popping up in wappalyzer on our prod build) as the methods don't seem be used anywhere obvious in the codebase

I'm not entirely sure to be honest, so if you or anyone has more insight here please let me know. I guess one easy way to check would be to run a build locally with moment, also run with one of the alternatives, and compare bundle sizes (I can do this today or tomorrow)

Reply:
it does indeed seem to affect bundle size. That said, our bundle is... very big. This might be worth discussing in an architecture call or devoting some time to. 

Without moment: 
<img width="478" alt="Screen Shot 2022-02-08 at 9 49 58 PM" src="https://user-images.githubusercontent.com/9052511/153124417-7ad11583-62cf-4877-9ffc-5e3f5dafe0e5.png">
And with: 
![image](https://user-images.githubusercontent.com/9052511/153124448-c2d85680-1d21-425f-a217-ff433570d41a.png)

Reply:
@mhagel I updated to dayjs instead of date-fns 👍